src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -49,3 +48,4 @@ IL3001 | Publish  | Warning  | DoNotUseAssemblyGetFilesInSingleFile, [Documentat
 Rule ID | New Category | New Severity | Old Category | Old Severity | Notes
 --------|--------------|--------------|--------------|--------------|-------
 CA2208 | Usage | Info | Usage | Hidden | InstantiateArgumentExceptionsCorrectlyAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2208)
+CA2012 | Reliability | Info | Reliability | Hidden | UseValueTasksCorrectlyAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2012)
```

Comment:
Given this rule has not shipped, it should be fine to just move this entry back up and change from 'Hidden' to 'Info'. Also can you please file a bug to ensure the code fix itself does an in-place replacement for such cases instead of marking it as a changed rule?

Reply:
Actually, did you manually change this file?

Reply:
I did manually change this file. Was that not the right thing to do?

Reply:
You answered below.